build.gradle

```diff
@@ -2075,8 +2075,7 @@ project(':connect:file') {
     compile libs.slf4jApi
 
     testCompile libs.easymock
-    testCompile libs.junitJupiterApi
-    testCompile libs.junitVintageEngine
+    testCompile libs.junitJupiter
     testCompile libs.powermockJunit4
```

Comment:
It seems power mock is not used by this module anymore. Could we remove them?

Reply:
Done. They are already removed now.